blockwarriors-frontend/package.json

```diff
@@ -38,6 +38,7 @@
     "@radix-ui/react-toggle": "^1.0.3",
     "@radix-ui/react-toggle-group": "^1.1.1",
     "@radix-ui/react-tooltip": "^1.0.7",
+    "@shadcn/ui": "^0.0.4",
```

Comment:
We already have radix-ui. Reason for shadcn?

Reply:
I'm unsure since it was automatically added to package.json at some point.